src/Semantic/CLI.hs

```diff
@@ -180,8 +180,22 @@ filePathReader = eitherReader parseFilePath
     parseFilePath arg = case splitWhen (== ':') arg of
```

Comment:
I don't love that we allow this on the cli, but didn't want to make breaking changes in this PR. I think in a future iteration we should remove this ability to specify a file language now that our file detection is a bit better.